pkg/config/ledger_config.go

```diff
@@ -19,6 +19,8 @@ type Ledger struct {
 	// SkipBlockVerification allows to disable verification of received
 	// blocks (including cryptographic checks).
 	SkipBlockVerification bool `yaml:"SkipBlockVerification"`
+	// SaveInvocations enables contract smart contract invocation data saving.
+	SaveInvocations bool `yaml:"SaveInvocations"`
```

Comment:
Please, adjust `docs/node-configuration.md`. Write some details on how this feature works and recommendations on situations when it should be enabled/disabled.